src/stores/oracleDetails.js

```diff
@@ -42,7 +39,7 @@ export const useOracleDetailsStore = defineStore('oracleDetails', () => {
   }
 
   async function fetchOracle() {
-    const { data } = await axios.get(`${MIDDLEWARE_URL}/v2/oracles/${oracleId.value}`)
+    const { data } = await axios.get(`${MIDDLEWARE_URL}/v2/oracles/${oracleId.value}?tx_hash=true`)
```

Comment:
as advised in here https://github.com/aeternity/ae_mdw/issues/1425#issuecomment-1700899383